arm-network/2016-09-01/swagger/network.json

```diff
@@ -3985,14 +3985,12 @@
             "type": "string",
             "description": "The name of the virtual network gateway."
           },
-          {
-            "name": "parameters",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/VirtualNetworkGateway"
-            },
-            "description": "Parameters supplied to the Begin Reset Virtual Network Gateway operation through Network resource provider."
+          { 
```

Comment:
Server now does not need this param:VirtualNetworkGateway  so I removed it. Powershell cmdlet params will not change, take a look at https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/ResetAzureVirtualNetworkGatewayCommand.cs